axum-macros/tests/from_request/pass/override_rejection_with_via_on_struct.rs

```diff
@@ -0,0 +1,41 @@
+use axum::{
+    extract::rejection::JsonRejection,
+    response::{IntoResponse, Response},
+    routing::get,
+    Router,
+};
+use axum_macros::FromRequest;
+use serde::Deserialize;
+
+fn main() {
+    let _: Router = Router::new().route("/", get(handler).post(handler_result));
+}
+
+#[derive(Deserialize)]
+struct Payload {}
+
+async fn handler(_: MyJson<Payload>) {}
+
+async fn handler_result(_: Result<MyJson<Payload>, MyJsonRejection>) {}
+
+#[derive(FromRequest)]
+#[from_request(
+    // TODO(david): `rejection(...)` without `via(...)`
```

Comment:
Don't forget your `TODO`s 🙂

Reply:
Oh yeah! I did actually fix this but forgot to remove the todo.